test/test_testing.py

```diff
@@ -1158,5 +1148,35 @@ def test_mismatching_values_msg_imag(self):
                 fn()
 
 
-if __name__ == '__main__':
+class TestAssertCloseQuantized(TestCase):
+    def test_mismatching_is_quantized(self):
+        actual = torch.tensor(1.0)
+        expected = torch.quantize_per_tensor(actual, scale=1.0, zero_point=0, dtype=torch.qint32)
+
+        for fn in assert_close_with_inputs(actual, expected):
+            with self.assertRaisesRegex(AssertionError, "is_quantized"):
+                fn()
+
```

Comment:
@jerryzh168 what is a good way to create a tensor with the `per_channel_affine` and `per_channel_affine_float_qparams` to test our implementation?

Reply:
you can use https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L5205 to create a per channel quantized Tensor, to create a per_channel_affine_float_qparams tensor I think you'll need to use quantize_per_channel for now: https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/utils.py#L18